src/extension/debug-page/types.ts

```diff
@@ -0,0 +1,24 @@
+export interface BackupFormat {
+    buildInfo: Record<string, string | undefined>
```

Comment:
I suggest using `Map` instead of `Record`. A primary cons is that `Map` can be used with `for ... of ...` with type.